packages/web-pkg/README.md

```diff
@@ -0,0 +1,17 @@
+# web-pkg
+
+`web-pkg` is a package that provides utilities that can be useful when creating and developing extensions for ownCloud Web.
+
+It's important to note that this package solely provides types and lacks an actual standalone implementation. This is because `web-pkg` is intended solely for Web extension development, and the real implementation is already provided by the Web runtime.
```

Comment:
```suggestion
It's important to note that this package solely provides types because `web-pkg` is intended solely for Web extension development. The actual implementation of the utilities is provided by the Web runtime automatically.
```